### PR TITLE
December 2021 Finance Ctte update mail

### PR DESCRIPTION
**Pull request by jdswinbank**

### CODE REVIEW

File: finance/community-updates/2021-11.txt [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Dear Astropy Community,
+
+THe Finance Committee would like to give you some updates on our
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:3]:
```suggestion
The Finance Committee would like to give you some updates on our
```

File: finance/community-updates/2021-11.txt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Project-wide infrastructure improvements, targeted work on specific
+areas of the code that are in need of support, enhancements to the our
+educational materials, and better support for affiliated packages. The
+full text (with some personal details redacted) is available at [1].
+
+The proposal was successful, and has provides a further $634,590 to
+support the fantastic work being undertaken on Astropy. The Finance
+Committee will arrange processes for disbursing the funds which are
+consistent with the goals of the proposal and based on the model we've
+adopting for assigning Moore funding over the past couple of years.
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:20]:
```suggestion
adopted for assigning Moore funding over the past couple of years.
```

File: finance/community-updates/2021-11.txt [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+successfully used the Moore funding to build a solid set of processes
+and a governance structure which enables us to obtain funding from more
+“traditional” academic sources. This is a great step towards ensuring
+the financial viability of the project into the future.
+
+[1] https://drive.google.com/file/d/18JsTRahNWnvign6ao6BP_OkINHnP8g3n/view
+
+# Gemini contract
+
+The Science User Support Department at Gemini Observatory has funded an
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:32]:
```suggestion
The Science User Support Department at Gemini Observatory has funded
```

File: finance/community-updates/2021-11.txt [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+Simon Conseil as an Astropy contractor to provide maintenance
+and development that benefits both the Astropy and DRAGONS projects on a
+part-time basis (averaging 5 hours per week over a period of a year or
+two). The details of the work are described in the SOW [2].
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:36]:
```suggestion
two). The details of the work are described in the statement-of-work[2].
```
Even I did not recognize the acronym until I check the link.

File: finance/community-updates/2021-11.txt [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+funding in future years. We welcome your feedback on how to continue to
+improve our approach to make this as transparent and effective as
+possible.
+
+[3] https://groups.google.com/g/astropy-dev/c/c1M2F7rZoF4/m/nbNdg1OsAAAJ
+
+# Meetings, workshops, and other travel
+
+We've supported project members to participate in and represent Astropy
+at a number of fantastic conferences and workshops over the last year.
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:63]:
```suggestion
at a number of conferences and workshops over the last year.
```
I know, I know, some hyperbole. But do we really want to judge if all those conference where really fantastic?

File: finance/community-updates/2021-11.txt [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+These include:
+
+- Three project members were funded to (vrirtually) attended ADASS XXX,
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:66]:
```suggestion
- Three project members were funded to (virtually) attended ADASS XXX,
```

File: finance/community-updates/2021-11.txt [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+- Three project members hosted the “Astropy and NumFocus Sponsored
+  Projects” booth in the Virtual Exhibit Hall at AAS 237, while also ran
+  two workshops: “Introduction to Astropy” and “Research with Astropy”. [5]
+
+- Similarly, three project members paricipated in AAS 238, again hosting
+  workshops entitled “Introduction to Astropy” and “Astronomical
+  Research with Astropy”. [6]
+
+- We also supported three project memnbers to participate in the SciPy
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:78]:
```suggestion
- We also supported three project members to participate in the SciPy
```

File: finance/community-updates/2021-11.txt [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+[5] https://github.com/astropy/astropy-project/issues/140
+[6] https://github.com/astropy/astropy-project/issues/201
+[7] https://github.com/astropy/astropy-project/issues/210
+
+# No longer an Interim Finance Committee
+
+One of the first acts of the recently-elected Coordination Committee
+(“CoCo3”) was to confirm the Finance Committee, with its current
+members, as a part of standing project organization and to remove the
+work “interim” from its title. We're now just the “Astropy Finance
```
Comment (hamogu) [inline finance/community-updates/2021-11.txt:94]:
```suggestion
“interim” from its title. We're now just the “Astropy Finance
```

**Review by hamogu — CHANGES_REQUESTED** [review]
An excellent summary. Really fantastic [*] work. 

[*] See comment in this review about "fantastic". Can't link to that comment yet, because it's not been posted yet.

File: finance/community-updates/2021-11.txt [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+In January of this year, several members of the project submitted a
+proposal to the NASA ROSES-2020 call, section E.7 (Support for Open
+Source Tools, Frameworks, and Libraries). The proposal included a mix of
+Project-wide infrastructure improvements, targeted work on specific
+areas of the code that are in need of support, enhancements to the our
+educational materials, and better support for affiliated packages. The
+full text (with some personal details redacted) is available at [1].
+
+The proposal was successful, and has provides a further $634,590 to
+support the fantastic work being undertaken on Astropy. The Finance
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:17]:
```suggestion
support the fantastic work being undertaken by the Astropy Project. The Finance
```

File: finance/community-updates/2021-11.txt [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+Committee will arrange processes for disbursing the funds which are
+consistent with the goals of the proposal and based on the model we've
+adopted for assigning Moore funding over the past couple of years.
+
+This is a huge milestone for the project: it shows that we've
+successfully used the Moore funding to build a solid set of processes
+and a governance structure which enables us to obtain funding from more
+“traditional” academic sources. This is a great step towards ensuring
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:25]:
```suggestion
“traditional” funding sources. This is a great step towards ensuring
```

File: finance/community-updates/2021-11.txt [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+[1] https://drive.google.com/file/d/18JsTRahNWnvign6ao6BP_OkINHnP8g3n/view
+
+# Gemini contract
+
+The Science User Support Department at Gemini Observatory has funded
+Simon Conseil as an Astropy contractor to provide maintenance
+and development that benefits both the Astropy and DRAGONS projects on a
+part-time basis (averaging 5 hours per week over a period of a year or
+two). The details of the work are described in the statement of work [2].
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:36]:
```suggestion
The Science User Support Department at Gemini Observatory has provided funding to the Astropy Project
for maintenance and development work that benefits both the Astropy and DRAGONS projects. We've enlisted Simon Conseil as a part-time contractor (averaging 5 hours per week over a period of a year or
two) to do this work. The details of the work are described in the statement of work [2].
```

File: finance/community-updates/2021-11.txt [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+We organized a process to select and distribute funds from the Moore
+grant to a variety of projects, suggested and selected by the project
+membership. We've announced results in detail in the past [3] so we
+won't reiterate them in detail here, but we're delighted to report that
+most many of the projects are now well underway.
+
+We except to organize similar processes to distribute Moore and NASA
+funding in future years. We welcome your feedback on how to continue to
+improve our approach to make this as transparent and effective as
+possible.
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:56]: Add something about the survey.

File: finance/community-updates/2021-11.txt [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+[3] https://groups.google.com/g/astropy-dev/c/c1M2F7rZoF4/m/nbNdg1OsAAAJ
+
+# Meetings, workshops, and other travel
+
+We've supported project members to participate in and represent Astropy
+at a number of conferences and workshops over the last year.
+These include:
+
+- Three project members were funded to (virtually) attended ADASS XXX,
+  where Brigitta Sipocz spoke on “astroquery: a community driven
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:67]:
```suggestion
- Three project members were funded to (virtually) attended ADASS XXXI,
  where Brigitta Sipocz spoke on “astroquery: a community driven
```

File: finance/community-updates/2021-11.txt [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+  collaboration of data providers and data consumers”. [4]
+
+- Three project members hosted the “Astropy and NumFocus Sponsored
+  Projects” booth in the Virtual Exhibit Hall at AAS 237, while also ran
+  two workshops: “Introduction to Astropy” and “Research with Astropy”. [5]
```
Comment (kelle) [inline finance/community-updates/2021-11.txt:72]:
```suggestion
- Three project members hosted the “Astropy and NumFocus Sponsored
  Projects” booth in the Virtual Exhibit Hall at AAS 237. Project members also provided
  two workshops: “Introduction to Astropy” and “Research with Astropy”. [5]
```

**Review by kelle — CHANGES_REQUESTED** [review]

File: finance/community-updates/2021-11.txt (hunk 13 above)
Comment (jdswinbank) [reply finance/community-updates/2021-11.txt:67]: No, it really was “XXX” — our last mail went out before that meeting. I'm not aware of us funding anybody to go to ADASS XXXI — if we did, we don't seem to have written an issue about it.

File: finance/community-updates/2021-11.txt (hunk 12 above)
Comment (jdswinbank) [reply finance/community-updates/2021-11.txt]: Done

**Pull request merged**